src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import bookReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+export const mergedReducers = combineReducers({
+  bookReducer,
+  categoriesReducer,
+});
+
+const store = createStore(mergedReducers);
+
+export default store;
```

Comment:
- [ ] [Optional] It would be nice to install and import some handy middleware like the redux logger. To install it, you can run this command: `npm i --save redux-logger`. You can read this [documentation](https://www.npmjs.com/package/redux-logger) to have an idea of how to implement it. 👍